src/main/java/com/puppycrawl/tools/checkstyle/checks/regexp/SinglelineDetector.java

```diff
@@ -66,6 +68,21 @@ public void processLines(List<String> lines) {
         finish();
     }
 
+    /**
+     * Processes a set of lines looking for matches.
+     * @param fileText {@link FileText} object contains the lines to process.
+     */
+    public void processLines(FileText fileText) {
```

Comment:
There is also a method `public void processLines(List<String> lines)`.
Why is the other method remaining? Shouldn't it be swapped for this new method instead of keeping both?